packages/optimistic-oracle/src/proposer.js

```diff
@@ -48,6 +54,16 @@ class OptimisticOracleProposer {
         isValid: x => {
           return x >= 6000000 && x < 15000000;
         }
+      },
+      disputePricePrecisionOfError: {
```

Comment:
this is a good way way deal with this.

I have a question about the best way to generalise this. When we talk about precision we are normally talking about the number of digits after the decimal point. however, in the case of really large numbers, it might be better to consider precision interns of the number of significant figures within the number.

For example, consider the number 12.3456789. If you said you wanted to round this to a precision of 6 would my understanding of your implementation is that it would turn this into 12.3456790, rounding to the 6th decimal place. 

How would you then round the number 123456789 to a precision of 6? based off your implementation I _think_ it would result in not modifying the number. however, there is an equal number of significant figures in this second case and you could argue that it should be rounded equally to 123456000. In this case we look for the number of significant figures and "round" to that. If this pattern was followed with the first example the rounding would have yielded 12.3456000, thereby considering all significant figures in the process, rather than just those after the decimal point. 

The point I'm trying to make is that we should ensure we converge on a consistant definition of precision for small and big numbers alike. To be clear: the proposal I'm making it to consider precision interns of significant figures, rather than decimal points. WDYT?

Reply:
This is a really good point and I think large numbers we'd be interested in rouding to a number left of the decimal point. The easyish fix for this is to replace `toFixed()` with `toPrecision()` in determining the `rounded` values. WDYT?

`toPrecision` sets the # of significant digits, which includes both left and right sides of the decimal

Reply:
Alternatively, we could just define the buffer as a percentage rather than a # of decimals of precision. Say we allow prices to differ by 10% before we'd dispute. WDYT?

Reply:
I implemented this change as I think its more flexible ^

Reply:
Good changes! I'm happy with how you modified this to be a percentage; this way it's not about decimal points/precision but rather an absolute error relative to the "true" value. I'm more happy with this! thank for changing it.